examples/js/renderers/CSS3DRenderer.js

```diff
@@ -95,28 +95,68 @@ THREE.CSS3DRenderer = function () {
 
 	}
 
-	function getCameraCSSMatrix( matrix ) {
+	function getCameraCSSMatrix( camera ) {
+
+		var elements = camera.matrixWorldInverse.elements;
+
+		if ( camera.isOrthographicCamera ) {
+
+			var sx = 2 / ( camera.right - camera.left ) * _widthHalf * camera.zoom;
+			var sy = 2 / ( camera.top - camera.bottom ) * _heightHalf * camera.zoom;
+
+			var tx = - ( ( camera.right + camera.left ) / ( camera.right - camera.left ) * _widthHalf ) * camera.zoom;
+			var ty = ( ( camera.top + camera.bottom ) / ( camera.top - camera.bottom ) * _heightHalf ) * camera.zoom;
+
+			// Multiplying matrixWorldInverse with this:
+			//
+			// sx  0  0  tx
+			//  0 sy  0  ty
+			//  0  0 sz  tz
+			//  0  0  0   1
+			//
+			// sz === 1, tz === 0
+
+			return 'matrix3d(' +
+				epsilon( sx * elements[ 0 ] ) + ',' +
+				epsilon( - sy * elements[ 1 ] ) + ',' +
+				epsilon( elements[ 2 ] ) + ',' +
+				'0,' +
```

Comment:
Please do not hardwire values here.
```javascript
epsilon( elements[ 3 ] ) + ',' +
```

Reply:
Why? You told me it's always 0, and it can be simplified.

Reply:
That's a separate optimization. If it were to be implemented in this case, it should be implemented consistently throughout the file.